dojson/utils.py

```diff
@@ -44,6 +44,21 @@ def wrapper(self, key, value, **kwargs):
     return wrapper
 
 
+def ignore_list_elem(f):
```

Comment:
I just followed the current code structure, to skip None return values you have the ignore_value above. IMO it's easier to use, as you don't have to know the implementation behind it (using exceptions), you can just rely on this decorator to do whatever is needed.